openslides_backend/action/actions/meeting/update.py

```diff
@@ -227,6 +228,19 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
             raise ActionException(
                 "A meeting cannot be locked from the inside and a template at the same time."
             )
+        organization = self.datastore.get(
+            ONE_ORGANIZATION_FQID, ["require_duplicate_from"]
+        )
+        if organization.get("require_duplicate_from"):
+            if set_as_template is not None and not has_organization_management_level(
+                self.datastore,
+                self.user_id,
+                OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION,
+            ):
+                raise ActionException(
+                    "A meeting cannot be set as a template by a committee manager, if duplicate from is required."
+                )
```

Comment:
see my comment in `meeting/clone.py`

Reply:
Use 'and' to combine the if-conditionals and rm the comma.